mozphab/jujutsu.py

```diff
@@ -29,6 +29,11 @@
 
 
 class Jujutsu(Repository):
+    @classmethod
+    def is_repo(cls, path: str) -> bool:
+        """Quick check for repository at specified path."""
+        return os.path.exists(os.path.join(path, ".jj"))
+
```

Comment:
nitpick: Moved this above the other named "blocks" of `def`s, since it doesn't really belong to any of those.